scientific-computing-with-python/learn-object-oriented-programming-by-building-a-coffee-making-machine/CoffeeMachine.py

```diff
@@ -0,0 +1,153 @@
+from time import sleep
+
+"""
+The Maker class represents the coffee maker itself. It keep tracks of the water, milk,
+and beans in the machine. Makes coffee and reports on the resources.
+"""
+
+
+class Maker:
+
+    def __init__(self):
+        self.resources = {
+            "water": 300,
+            "milk": 200,
+            "beans": 100,
+        }
+
+    def report(self):
+        print(f"Water: {self.resources['water']}ml")
+        print(f"Milk: {self.resources['milk']}ml")
+        print(f"Coffee: {self.resources['beans']}g")
```

Comment:
```suggestion
        print(f"Water: {self.resources['water']} ml")
        print(f"Milk: {self.resources['milk']} ml")
        print(f"Coffee: {self.resources['beans']} g")
```
Adding some space to the output